python/pycylon/pycylon/api/lib.pxd

```diff
@@ -54,34 +54,42 @@ cdef api shared_ptr[CCylonContext] pycylon_unwrap_context(object context)
 
 cdef api shared_ptr[CMPIConfig] pycylon_unwrap_mpi_config(object config)
 
-cdef api shared_ptr[CTable] pycylon_unwrap_table (object table)
+cdef api shared_ptr[CTable] pycylon_unwrap_table(object table)
 
-cdef api shared_ptr[CDataType] pycylon_unwrap_data_type (object data_type)
+cdef api shared_ptr[CDataType] pycylon_unwrap_data_type(object data_type)
 
 cdef api CCSVReadOptions pycylon_unwrap_csv_read_options(object csv_read_options)
 
 cdef api CCSVWriteOptions pycylon_unwrap_csv_write_options(object csv_write_options)
 
-cdef api CSortOptions* pycylon_unwrap_sort_options(object sort_options)
+cdef api CSortOptions * pycylon_unwrap_sort_options(object sort_options)
 
 cdef api shared_ptr[CBaseArrowIndex] pycylon_unwrap_base_arrow_index(object base_arrow_index)
 
 cdef api CType pycylon_unwrap_type(object type)
 
 cdef api CLayout pycylon_unwrap_layout(object layout)
 
-cdef api CJoinConfig* pycylon_unwrap_join_config (object config)
+cdef api CJoinConfig * pycylon_unwrap_join_config(object config)
 
-cdef api object pycylon_wrap_table(const shared_ptr[CTable] &ctable)
+cdef api object pycylon_wrap_table(const shared_ptr[CTable] & ctable)
```

Comment:
nit: Did this happen by the code formatter or are we going to put it like that. The existing one seems to be more intuitive though.

Reply:
yeah, its the stupid formatter! :-/